### PR TITLE
Fix Docker/SES stuff

### PR DESCRIPTION
**Pull request by duner**

### CODE REVIEW

File: config/environments/production.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@
   # config.action_dispatch.x_sendfile_header = 'X-Accel-Redirect' # for NGINX
 
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
-  config.force_ssl = true
+  config.force_ssl = false
```
Comment (ivarvong) [inline config/environments/production.rb]: Why?

**Comment by duner** [comment]
Ahh oops — meant to make this as a PR to our fork instead of to the main project — I can close for now, but there will be a PR at some point that fixes some of this stuff + adds more documentation.

File: config/environments/production.rb (hunk 1 above)
Comment (duner) [inline config/environments/production.rb]: We had some problems getting Docker to work locally over https, so we modified this for the time being while we were figuring out how Klaxon works locally. (See my other comment about this PR being very much in progress.)

**Comment by tommeagher** [comment]
Hey @duner, sorry for the mixup. But once you're all finished and want to merge the changes back to the repo, we'd love the contribution. And we'll be eager to hear feedback about how Klaxon is working for y'all. Say hi to @geraldarthur for me.

**Comment by tommeagher** [comment]
Closing this for now while @geraldarthur et al. work on their. Will happily accept another PR down the road if you want to share back.

**Comment by geraldarthur** [comment]
Whoops @tommeagher! Didn't realize this fellow was still open. Thanks for closing it out and I can send over a new PR soon. Also, you may direct all color theory inquires to @andymboyle

**Pull request closed without merging**